spec/fixtures/swift/default/Shop.swift

```diff
@@ -0,0 +1,22 @@
+/* DO NOT EDIT | Generated by gyro */
+
+import RealmSwift
+import Foundation
+
+final class Shop: Object {
+
+  enum Attributes: String {
+    case active = "active"
+    case isOpened = "isOpened"
```

Comment:
Just because I like nitpicking: `isOpened` is not correct English 😜 `isOpen` would be more correct 😉